commit 714543011f57b6427e16a1be1d9e4cd3e018b241
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Sun Aug 21 17:24:58 2022 +0200

    get rid of zImage.pseries
    
    This fails to build and we don't care about any of the wrappers.

diff --git a/arch/powerpc/boot/Makefile b/arch/powerpc/boot/Makefile
index a9cd2ea..1e514dd 100644
--- a/arch/powerpc/boot/Makefile
+++ b/arch/powerpc/boot/Makefile
@@ -267,10 +267,7 @@ quiet_cmd_wrap	= WRAP    $@
 		$(CROSSWRAP) $(if $3, -s $3)$(if $4, -d $4)$(if $5, -i $5) \
 		vmlinux
 
-image-$(CONFIG_PPC_PSERIES)		+= zImage.pseries
-image-$(CONFIG_PPC_POWERNV)		+= zImage.pseries
 image-$(CONFIG_PPC_MAPLE)		+= zImage.maple
-image-$(CONFIG_PPC_IBM_CELL_BLADE)	+= zImage.pseries
 image-$(CONFIG_PPC_PS3)			+= dtbImage.ps3
 image-$(CONFIG_PPC_CHRP)		+= zImage.chrp
 image-$(CONFIG_PPC_EFIKA)		+= zImage.chrp
@@ -452,7 +449,7 @@ $(obj)/zImage.initrd:	$(addprefix $(obj)/, $(initrd-y))
 # anything not in $(targets)
 clean-files += $(image-) $(initrd-) cuImage.* dtbImage.* treeImage.* \
 	zImage zImage.initrd zImage.chrp zImage.coff zImage.holly \
-	zImage.miboot zImage.pmac zImage.pseries \
+	zImage.miboot zImage.pmac \
 	zImage.maple simpleImage.* otheros.bld
 
 # clean up files cached by wrapper
